plugins/kotlin/idea/src/org/jetbrains/kotlin/idea/quickfix/createFromUsage/createVariable/CreateParameterFromUsageFix.kt

```diff
@@ -92,6 +92,7 @@ class CreateParameterFromUsageFix<E : KtElement>(
             callableInfosFactory: (E) -> List<CallableInfo>?
         ): CreateParameterFromUsageFix<E> = CreateParameterFromUsageFix(element, dataProvider = fun(element): CreateParameterData<E>? {
             val info = callableInfosFactory.invoke(element)?.singleOrNull().safeAs<PropertyInfo>() ?: return null
+            if (info.receiverTypeInfo.staticContextRequired) return null
```

Comment:
I failed to find any test which covers this `if`. Are there any such tests? Or if this `if` isn't more required then can you please elaborate why it's not required (and this case let's remove this `if`) or if it's required then please add tests that cover this `if`

Reply:
If this `if` statement is removed, the following tests will fail.

- testData/quickfix/createFromUsage/createClass/referenceExpression/enumEntryWithQualifier.kt
- testData/quickfix/createFromUsage/createClass/referenceExpression/classWithQualifier.kt